mlir/lib/ExecutionEngine/ROCm/rocm-runtime-wrappers.cpp

```diff
@@ -279,6 +279,36 @@ extern "C" void mgpuMemCopy(float *sourceAllocated, float *sourceAligned,
             static_cast<hipMemcpyKind>(copyDirection));
 }
 
+extern "C" StridedMemRefType<int32_t, 1>
```

Comment:
are these library routines added because in_warp_transpose takes i32?

Reply:
(Well, mainly so I can have an integer test instead of a float one)